examples/experimental/onnx/ac_configs/efficientnet-b0_imagenet_onnx.yml

```diff
@@ -0,0 +1,42 @@
+models:
+  - name: resnet50_imagenet_int8_per_tensor
+    launchers:
+      - framework: onnx_runtime
+        #device: CPU
+        adapter: classification
+        execution_providers: ['OpenVINOExecutionProvider']
+        inputs:
+          - name: data
+            type: INPUT
+            shape: [1,3,224,224]
+
+    datasets:
+      - name: imagenet_1000_classes
+        data_source: /home/aleksei/datasets/ILSVRC2012_img_val/
```

Comment:
I think we shouldn't place here paths to the locally placed data.

Reply:
Done